MassOS isn't one of the "approved" distros that the Polkit build system can
detect, and when it fails to detect the correct distro, the configuration files
for PAM authentication are set incorrectly, completely breaking any and all
functionality of the package. This patch sets the "undetected distro" values to
the ones needed for MassOS. It also removes an annoying warning, because we
KNOW it can't detect our distro (that's why we created this patch).

--- a/meson.build
+++ b/meson.build
@@ -303,10 +303,6 @@
       break
     endif
   endforeach
-
-  if os_type == ''
-    message('Linux distribution autodetection failed, specify the distribution to target using -Dos_type=')
-  endif
 endif
 
 pam_include = get_option('pam_include')
@@ -336,9 +332,9 @@
   else
     pam_conf = {
       'PAM_FILE_INCLUDE_AUTH': 'system-auth',
-      'PAM_FILE_INCLUDE_ACCOUNT': 'system-auth',
-      'PAM_FILE_INCLUDE_PASSWORD': 'system-auth',
-      'PAM_FILE_INCLUDE_SESSION': 'system-auth',
+      'PAM_FILE_INCLUDE_ACCOUNT': 'system-account',
+      'PAM_FILE_INCLUDE_PASSWORD': 'system-password',
+      'PAM_FILE_INCLUDE_SESSION': 'system-session',
     }
   endif
 else
